var/spack/repos/builtin/packages/freecad/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freecad(CMakePackage):
+    """FreeCAD is an open-source parametric 3D modeler made primarily
+    to design real-life objects of any size. Parametric modeling
+    allows you to easily modify your design by going back into your
+    model history to change its parameters."""
+
+    homepage = "https://www.freecad.org/"
+    url = "https://github.com/FreeCAD/FreeCAD/archive/refs/tags/0.20.2.tar.gz"
+
+    maintainers("aweits")
+
+    version("0.20.2", sha256="46922f3a477e742e1a89cd5346692d63aebb2b67af887b3e463e094a4ae055da")
+
+    depends_on("opencascade")
+    depends_on("xerces-c")
+    depends_on("vtk")
+    depends_on("salome-med")
+    depends_on(
+        "boost+python+filesystem+date_time+graph+iostreams+program_options+regex+serialization+system+thread"  # noqa: E501
+    )
+    depends_on("qt@5:")
+    depends_on("swig")
+    depends_on("netgen")
+    depends_on("pcl")
+    depends_on("coin3d")
+    depends_on("python")
+    depends_on("gmsh+opencascade")
+    depends_on("py-pyside2@1.2.4:", type=("build", "run"))
+    depends_on("py-matplotlib@3.0.2:", type=("build", "run"))
+    depends_on("py-six@1.12.0:", type=("build", "run"))
+    depends_on("py-markdown@3.2.2:", type=("build", "run"))
+    depends_on("py-pivy", type=("build", "run"))
+    depends_on("py-pybind11", type=("build", "run"))
```

Comment:
pybind11 is often build or build/link, not often run. Any idea how it's used? Also, where did you get the list of deps? Might help me review.

Reply:
list of deps was mostly from cmake, there's a requirements.txt, but mostly got them from compiling/testing/working with example input files. probably some future work to do in expanding the python deps especially.